.travis.yml

```diff
@@ -33,6 +34,8 @@ matrix:
       env: DJANGO_VERSION="1.10"
     - python: "3.3"
       env: DJANGO_VERSION="1.11"
+    - python: "3.3"
```

Comment:
Why would we need Python 3.3 for Django 2.0?
https://docs.djangoproject.com/en/2.0/releases/2.0/#python-compatibility

Reply:
@fengsi We could drop it from the testing matrix to be excluded for Django 2.0

Reply:
@fengsi we don't, this is an `exclude` list ;)

Reply:
@fengsi If you look on [line 26](https://github.com/zapier/django-rest-hooks/blob/master/.travis.yml#L26). Python 3.3 is added as part of the exclusion for Django 2.0

Reply:
my bad 👍